client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -160,13 +176,21 @@ function PaymentMethodAdd( { selectedSite }: { selectedSite?: SiteDetails | null
 		} else {
 			page( partnerPortalBasePath( '/payment-methods' ) );
 		}
-	}, [ returnQueryArg, products, reduxDispatch ] );
+	}, [ returnQueryArg, products, isSiteCreationFlow, reduxDispatch ] );
 
 	useEffect( () => {
 		if ( paymentMethodRequired ) {
 			return;
 		}
 
+		// If this is a site creation flow, we will need to resume on the creation of site.
+		if ( isSiteCreationFlow ) {
+			dispatch( infoNotice( translate( 'A new WordPress.com site is on the way!' ) ) );
```

Comment:
I don't see this notice being displayed. Also, is this needed? We already show the below:

<img width="1422" alt="Screenshot 2023-12-20 at 1 25 37 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/ed6b96df-470d-44a5-8fdc-359a079302ce">

Reply:
Good catch! I have removed it as it is not necessary for this case. 👍